openstack/dds/v3/instances/requests.go

```diff
@@ -237,6 +241,66 @@ type EnabledOpts struct {
 	Enabled *bool `json:"enabled" required:"true"`
 }
 
+type AvailabilityZoneOpts struct {
+	TargetAzs string `json:"target_azs" required:"true"`
+}
+
+type DescriptionOpts struct {
+	Remark string `json:"remark" required:"true"`
+}
+
+// UpdateAvailabilityZone is a method to update the AvailabilityZone.
+func UpdateAvailabilityZone(c *golangsdk.ServiceClient, instanceId string, az string) (*AvailabilityZoneResp, error) {
+	opts := AvailabilityZoneOpts{
+		TargetAzs: az,
+	}
```

Comment:
it's better to this part in the provider

Reply:
done